File: PrintAlternates.py
def getAlternates(arr):
    res = []
    # Lặp qua các phần tử cách nhau một vị trí
    for i in range(0, len(arr), 2):
        res.append(arr[i])
    return res

if __name__ == "__main__":

    n = int(input("Nhập số lượng phần tử của mảng: "))
    # Khởi tạo mảng rỗng và nhập các phần tử
    arr = []
    print(f"Nhập {n} phần tử của mảng:")
    for i in range(n):
        num = int(input(f"Phần tử thứ {i + 1}: "))
        arr.append(num)
    
    res = getAlternates(arr)
    print("Các phần tử cách nhau một vị trí:", " ".join(map(str, res)))

File: PrintAlternates.c
#include <stdio.h>
void getAlternates(int arr[], int n) {
    // Lặp qua các phần tử có chỉ số chẵn
    for (int i = 0; i < n; i += 2) {
        printf("%d ", arr[i]);
    }
}
int main() {
    int n;
    printf("Nhập số lượng phần tử của mảng: ");
    scanf("%d", &n);
    int arr[n];
    // Yêu cầu nhập các phần tử của mảng
    printf("Nhập %d phần tử của mảng:\n", n);
    for (int i = 0; i < n; i++) {
        printf("Phần tử thứ %d: ", i + 1);
        scanf("%d", &arr[i]);
    }
    printf("Các phần tử cách nhau một vị trí: ");
    getAlternates(arr, n);

    return 0;
}
